<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsChange</class>
 <widget class="QDialog" name="SettingsChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>287</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="defaultLabel">
      <property name="text">
       <string>Default Mask:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="defaultMask"/>
    </item>
    <item>
     <widget class="QLabel" name="currentLabel">
      <property name="text">
       <string>Current Mask:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="currentMask"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>New Mask:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="newMaskEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="defaultButton">
      <property name="text">
       <string>Use Default</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
